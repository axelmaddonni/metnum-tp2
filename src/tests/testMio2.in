0 0.85 0 tests/testMio2.txt 0.00001
