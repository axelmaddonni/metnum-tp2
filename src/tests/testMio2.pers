0.5
2 4
3 18
4 1
6 35
