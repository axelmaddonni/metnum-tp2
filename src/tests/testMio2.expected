0.342466
0.131915
0.267872
0.18798
0.0348837
0.0348837
